types/stream-chat/index.d.ts

```diff
@@ -116,6 +116,11 @@ export interface ReactionResponse extends Reaction {
   created_at: string;
   updated_at: string;
 }
+
+export interface MuteParams {
+  user_id?: string;
```

Comment:
@ferhatelmas 50% chance you use the wrong field imo; what about we call one `userId` and `bannedByUserId`?

Reply:
I'm not clear completely. Do you suggest renaming properties to `userId` and `bannedByUserId`?